examples/rest_08_oauth_test.py

```diff
@@ -0,0 +1,108 @@
+"""
+REST OAuth.
+Minimal example to create a client and test OAuth
+"""
+#%%
+import threading
+import time
+from datetime import datetime
+from ibind import IbkrClient
+from ibind import var
+
+import pandas as pd
+from ibind.support.oauth import req_live_session_token, generate_oauth_headers, prepare_oauth
+
+client = IbkrClient(use_oauth=True,cacert=False)
+
+# print live session token
+print(f'live_session_token: {client.live_session_token}')
+# print access token
+print(f'live_session_token_expires_ms: {client.live_session_token_expires_ms}')
+
+#%%
+
+
+# Function that contains the while loop
+def background_task(client):
```

Comment:
Do we need to run this background task here? IBeam already calls `tickle` every minute. Is there any other reason?

Reply:
Ibeam starts the localhost and keeps it open, OAuth calls the ibkr oauth api and they use different base urls.
When I'm using oauth I don't start IBeam.
So I think they are completely different and a background tickle is useful.

Reply:
Sorry, my brain froze for a minute there. Naturally, we're not using IBeam, you're right. So we need to do this. I think it may be useful to implement it in IbkrClient. I'll take care of it, thank you for pointing it out 👏

Reply:
No problem, we're on the same page now.
I'll let you finish the tickle function off.